### PR TITLE
Change the PUT request handler to an even older way

### PR DESCRIPTION
**Pull request by RonaldTesting**
Hey, the way to handle PUT is modified.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -20,5 +20,7 @@ app.delete('/', (req, res)=>{
 });
 
 app.put('/', (req, res)=>{
-  res.send('Hello, PUT!');
```
Comment (RonaldChao) [inline app.js]: Hey why though?

File: app.js [hunk 2]
```diff
@@ -20,5 +20,7 @@ app.delete('/', (req, res)=>{
...
+  //res.send('Hello, PUT!');
+  res.send('Hey, PUT should be done this way!');
```
Comment (RonaldChao) [inline app.js]: Should be done in my way

Comment (RonaldTesting) [reply app.js]: Fine, do it your way

File: app.js (hunk 1 above)
Comment (RonaldTesting) [reply app.js]: I insist

Comment (RonaldChao) [reply app.js]: good

File: app.js (hunk 2 above)
Comment (RonaldChao) [reply app.js]: modify it please

File: app.js [hunk 3]
```diff
@@ -21,6 +21,6 @@ app.delete('/', (req, res)=>{
 
 app.put('/', (req, res)=>{
   //res.send('Hello, PUT!');
-  res.send('Hey, PUT should be done this way!');
+  res.send('Alright, do it the old way!');
```
Comment (RonaldChao) [inline app.js]: Good, It would be better if it's even older...haha

Comment (RonaldTesting) [reply app.js]: Fine.

**Pull request merged**